src/sinks/influxdb/logs.rs

```diff
@@ -592,24 +592,32 @@ mod tests {
         event.as_mut_log().insert("value", 100);
         event.as_mut_log().insert("timestamp", ts());
 
-        let sink = create_sink(
+        let mut sink = create_sink(
             "http://localhost:9999",
             "my-token",
             ProtocolVersion::V2,
             "vector",
-            ["metric_type"].to_vec(),
+            [].to_vec(),
         );
+        sink.transformer
+            .set_except_fields(Some(vec!["metric_type".into()]))
+            .unwrap();
```

Comment:
exclude default `metric_type` tag, so no tags present in final event.

Reply:
These comments would be useful in the code to explain what is being tested.